src/parsing.jl

```diff
@@ -12,7 +12,8 @@ immutable ResponseParser
         settings = ParserSettings(on_message_begin_cb, on_url_cb,
                           on_status_complete_cb, on_header_field_cb,
                           on_header_value_cb, on_headers_complete_cb,
-                          on_body_cb, on_message_complete_cb)
```

Comment:
Assuming not everyone is interested in the additional callbacks, this should probably be fixed in `HttpParser.jl` instead (i.e. https://github.com/JuliaWeb/HttpParser.jl/pull/49 with a non-deprecated version)